CcWorks/Workers/ReviewWorker.cs

```diff
@@ -56,12 +89,20 @@ public static async Task DoWork(ReviewCommandSettings settings, CommonSettings c
             var issue = await jira.Issues.GetIssueAsync(m.Groups[1].Value);
             Console.WriteLine("done");
 
-            if (!issue.Status.ToString().Equals("In progress", StringComparison.InvariantCultureIgnoreCase))
+            var issueStatus = issue.Status.ToString();
+
+            if (!issueStatus.Equals("In progress", StringComparison.InvariantCultureIgnoreCase))
             {
+                if (issueStatus.Equals("Ready for review", StringComparison.OrdinalIgnoreCase) && settings.AssignReviewLabel)
```

Comment:
This code looks strange. Assign label and throw exception that we don't work with this ticket. 
Looks like it's some debug or test code. Please remove it.